src/GravityFormsExtensions.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace P4\MasterTheme;
+
+/**
+ * Class P4\MasterTheme\GravityFormsExtensions
+ * The Gravity form plugin extension, use to add custom functionality like Planet4 form type.
+ */
+class GravityFormsExtensions {
+	/**
+	 * @var string The default gravity form type.
+	 */
+	public const DEFAULT_GF_TYPE = 'other';
+
+	/**
+	 * @var array The Planet4 Gravity form types.
+	 */
+	public const P4_GF_TYPES = [
+		[
+			'label' => 'Other',
+			'value' => 'other',
```

Comment:
What if you set the `value` to `DEFAULT_GF_TYPE`?

Reply:
The `DEFAULT_GF_TYPE` value belongs to `P4_GF_TYPES` array, so if we want to change the default value in future currently we need to update it at [one place](https://github.com/greenpeace/planet4-master-theme/pull/1763/files#diff-6c7e775b58a23510b6ac23bca880e6c85992c0ca431a51592bc53611aa775ec6R13) , if we use same constant in `P4_GF_TYPES` array, we need to changes on 2 places.